filebeat/module/elasticsearch/audit/test/test-audit-docker.log-expected.json

```diff
@@ -8,7 +8,7 @@
         "event.action": "anonymous_access_denied",
         "event.dataset": "elasticsearch.audit",
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
Similar to my other comment: https://github.com/elastic/beats/pull/13874/files#r330526232.